### DISCUSSION TITLE
The display on GitHub pages is different from my browser

### DISCUSSION THREAD

**Discussion by bruno-gurgel** [community/community#23667] [Other Feature Feedback, Questions, & Ideas]
<p>Hi! I’ve recently made a github page (<a href="https://bmg1612.github.io/blog-udacity/" rel="noopener nofollow ugc">https://bmg1612.github.io/blog-udacity/</a>) but when I open the project on my phone or Ipad, the card items off the home page are not separated. On the desktop is perfect!</p>
<p>I’ve tried to add margin, change the flex layout, but the problem continues. Maybe it is not importing the <span class="mention">@media</span> queries (but in that case would mess up the desktop version too)</p>
<p><a href="/uploads/short-url/cQWbCBxA9xvJOH67ChHOnCiiLMC.jpeg">IMG_3151.PNG|231x500</a></p>
<p>Can you guys help me? thanks!</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/bmg1612">@Bmg1612</a>, welcome to the GitHub Support Community!</p>
<p>It looks like that URL is showing a 404 now. Has your project moved and do you still need help with this, or have you managed to figure it out?</p>

**Comment by bruno-gurgel** [comment, marked as answer]
<p>Hi <a class="mention" href="/thomasshaped">@thomasshaped</a> ! Thanks for the reply. I figured it out, I actually initialized GitHub Pages in a wrong way. Fixed it already, thanks!</p>
